README.md

```diff
@@ -6,15 +6,17 @@
 rustup default nightly
 
 rustup component add rust-src
-rustup component add rustfmt-preview
+cargo install --force rustfmt-nightly
 
 rustup target add thumbv7em-none-eabihf
 
 cargo install svd2rust
 
 # currently used for linking
 # TODO - switch over to lld
-sudo apt-get install gcc-arm-none-eabi
+sudo add-apt-repository ppa:team-gcc-arm-embedded/ppa
```

Comment:
Just out of curiosity, were you running into issues with your distribution's package or after some new feature(s)?

Reply:
The version in ubuntu (18.04) is too old; the linker script in the arm-cortex-rt project requires some pretty new features.